### PR TITLE
chore: Python ingress changes

### PR DESCRIPTION
**Pull request by tiwarishubham635**
# Fixes #

A short description of what this PR does.

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] Run `make test-docker`
- [ ] Verify affected language:
    - [ ] Generate [twilio-go](https://github.com/twilio/twilio-go) from our [OpenAPI specification](https://github.com/twilio/twilio-oai) using the [build_twilio_go.py](./examples/build_twilio_go.py) using `python examples/build_twilio_go.py path/to/twilio-oai/spec/yaml path/to/twilio-go` and inspect the diff
    - [ ] Run `make test` in `twilio-go`
    - [ ] Create a pull request in `twilio-go`
    - [ ] Provide a link below to the pull request
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-oai-generator/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

**Review by manisha1997 — CHANGES_REQUESTED** [review]

File: src/test/java/com/twilio/oai/TwilioGeneratorTest.java [hunk 1]
```diff
@@ -49,6 +42,8 @@ public static void setUp() {
     public void launchGenerator() {
         final String pathname = "examples/spec/twilio_api_v2010.yaml";
 //        final String pathname = "examples/twilio_messaging_bulk_v1.yaml";
+//        final String pathname = "/Users/stiwari/di/github/twilio-oai/spec/yaml/twilio_messaging_bulk_v1.yaml";
```
[the remark this answers is not in the document]
Comment (tiwarishubham635) [reply src/test/java/com/twilio/oai/TwilioGeneratorTest.java:45]: Oh yeah, I will remove them

File: src/main/java/com/twilio/oai/resolver/python/PythonCodegenModelContainerDataTypeResolver.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import com.twilio.oai.resolver.common.CodegenModelContainerDataTypeResolver;
+import com.twilio.oai.resolver.csharp.OperationStore;
+import org.openapitools.codegen.CodegenModel;
+import org.openapitools.codegen.CodegenProperty;
+
+import java.util.List;
+import java.util.Stack;
+import java.util.regex.Pattern;
+
+public class PythonCodegenModelContainerDataTypeResolver extends CodegenModelContainerDataTypeResolver {
```
Comment (manisha1997) [inline src/main/java/com/twilio/oai/resolver/python/PythonCodegenModelContainerDataTypeResolver.java:15]: Add javadoc

**Review by manisha1997 — CHANGES_REQUESTED** [review]

File: src/main/java/com/twilio/oai/resolver/python/PythonCodegenModelDataTypeResolver.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+import com.twilio.oai.resolver.common.CodegenModelDataTypeResolver;
+import com.twilio.oai.resolver.common.CodegenModelResolver;
+import org.openapitools.codegen.CodegenModel;
+import org.openapitools.codegen.CodegenProperty;
+
+import java.util.Map;
+import java.util.function.BiConsumer;
+
+// Overriding default behavior and handling the enum
+public class PythonCodegenModelDataTypeResolver extends CodegenModelDataTypeResolver {
```
Comment (manisha1997) [inline src/main/java/com/twilio/oai/resolver/python/PythonCodegenModelDataTypeResolver.java:17]: https://github.com/twilio/twilio-oai-generator/pull/548#discussion_r1415125682

File: src/main/java/com/twilio/oai/resolver/python/PythonCodegenModelResolver.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+import com.twilio.oai.resolver.common.CodegenModelResolver;
+import org.openapitools.codegen.CodegenModel;
+import org.openapitools.codegen.CodegenProperty;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+public class PythonCodegenModelResolver extends CodegenModelResolver {
```
Comment (manisha1997) [inline src/main/java/com/twilio/oai/resolver/python/PythonCodegenModelResolver.java:17]: https://github.com/twilio/twilio-oai-generator/pull/548#discussion_r1415125682

**Review by manisha1997 — CHANGES_REQUESTED** [review]

**Review by kridai — APPROVED** [review]

**Review by shrutiburman — APPROVED** [review]

**Comment by tiwarishubham635** [comment]
C# tests are failing as sonar requires minimum java 17 version. See [this](https://docs.sonarsource.com/sonarqube/latest/requirements/prerequisites-and-overview/)

**Pull request merged**